client/src/main/kotlin/no/nav/su/se/bakover/client/stubs/person/PersonOppslagStub.kt

```diff
@@ -16,6 +16,7 @@ import java.time.LocalDate
 
 data class PersonOppslagStub(
     val dødsdato: LocalDate? = 21.januar(2021),
+    val fødselsdato: LocalDate = 1.januar(1990),
 ) : PersonOppslag {
 
     fun nyTestPerson(fnr: Fnr) = Person(
```

Comment:
Denne er jo reservert for test, så man kan tenke seg at man sender inn dødsdato og fødselsdato her og heller har en private val personer = Atomic<Pair<Fnr,Person>> - kanskje i tillegg til nyTestPerson for å gjøre overgangen enklere.

Også kan det man sender inn i konstruktøren heller være defaultPersonen?